setup/install_upstream_rust_keylime/test.sh

```diff
@@ -25,11 +26,25 @@ rlJournalStart
                 [ -f ${FILE} ] && rlRun "sed -i 's%/sys/kernel/security/tpm0/binary_bios_measurements%${TPM_BINARY_MEASUREMENTS}%' $FILE"
             done
         fi
+
+        if [ "$KEYLIME_RUST_CODE_COVERAGE" == "1" -o "$KEYLIME_RUST_CODE_COVERAGE" == "true" ]; then
+            rlRun "curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --default-toolchain none -y"
+            rlRun "source \"$HOME/.cargo/env\""
+            rlRun "rustup default nightly"
+            rlRun "rustup component add llvm-tools-preview"
+            sleep 3
+            #install parser for code coverage files
+            rlRun "cargo install grcov"
+            # -Z is deprecated, use -C
+            rlRun "export RUSTFLAGS='-Cinstrument-coverage'"
+        fi
+        #build
         rlRun "cargo build"
+
         rlAssertExists target/debug/keylime_agent
         [ -f /usr/local/bin/keylime_agent ] && rlRun "mv /usr/local/bin/keylime_agent /usr/local/bin/keylime_agent.backup"
         rlRun "cp target/debug/keylime_agent /usr/local/bin/keylime_agent"
-        if [ -n "${RUST_IMA_EMULATOR}" ]; then
+        if [ -n "${RUST_IMA_EMULATOR}" ] || [ -n "$KEYLIME_RUST_CODE_COVERAGE" ]; then
```

Comment:
I get the point, still not sure if it is really needed to modify this condition.